package com.aleftorres.curso.services.exceptions;

public class ResourceNotFoundException extends RuntimeException {
    public ResourceNotFoundException(Object id) {
        super("Recurso n√£o encontrado. ID " + id);
    }
}
